dawn/test/integration-test/from-sir/TestComputeMaximumExtent.cpp

```diff
@@ -12,35 +12,36 @@
 //
 //===------------------------------------------------------------------------------------------===//
 
-#include "dawn/Compiler/Driver.h"
-#include "dawn/IIR/ASTStmt.h"
-#include "dawn/IIR/StencilInstantiation.h"
-#include "dawn/SIR/SIR.h"
-#include "dawn/Serialization/SIRSerializer.h"
-#include "dawn/Support/FileSystem.h"
-#include <fstream>
-#include <gtest/gtest.h>
-#include <streambuf>
-#include <string>
+#include "TestFromSIR.h"
 
-using namespace dawn;
+namespace dawn {
```

Comment:
I think our other tests are in the anonymous namespace. Is there a reason this was added inside the dawn namespace?

Reply:
No particular reason, no. Changed.